policyengine_us/parameters/gov/states/mt/tax/income/subtractions/tuition/cap.yaml

```diff
@@ -0,0 +1,13 @@
+description: Montana caps the tuition subtraction at this maximum amount.
+values:
+  2022-01-01: 3_000
+metadata:
+  unit: currency-USD
+  period: year
+  label: Montana tuition subtraction cap
+  reference:
+  - title: 2023 MT Legal Code, Title 15, Section 30, Part 21, (5a)
+    href: https://leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0200/0150-0300-0210-0200.html
+  - title: 2022 MT Legal Code, Title 15, Section 30, Part 21, (5a)
+    href: https://leg.mt.gov/bills/2022/mca/title_0150/chapter_0300/part_0210/section_0200/0150-0300-0210-0200.html
```

Comment:
this seems to be 2021

we can add the 2022 and 2021 tax forms as well for reference

Reply:
Ok good point. However when I went to this link: https://leg.mt.gov/statute/ and clicked "Past MCA" they didn't have 2022 option. Only 2021 was available and years before then.

Reply:
we also need the tax form, not just the legal code
2022: https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf
2021: https://mtrevenue.gov/wp-content/uploads/mdocs/form%202%202021.pdf